src/google_map.js

```diff
@@ -611,7 +611,7 @@ export default class GoogleMap extends Component {
                 onChildMouseEnter={this_._onChildMouseEnter}
                 onChildMouseLeave={this_._onChildMouseLeave}
                 geoService={this_.geoService_}
-                projectFromLeftTop
+                insideMapPanes
```

Comment:
Sorry for being slow to respond.

[MapPanes](https://developers.google.com/maps/documentation/javascript/reference/3.exp/#overlay-view) is a concept in the Maps JS API. You can see that the react markers are placed inside the panes here:
https://github.com/google-map-react/google-map-react/blob/2cfb0a64f5093a8eca3b626b10e7dc0fecaef7cb/src/google_map.js#L599

Initially, the panes are located at the centre of the map. But as you pan the map, the panes move too, so that it's not necessarily (usually) to reposition the markers. As you zoom the map, you will have to reposition the markers.

So there are really three different reference points for coordinate: the map centre, the map top left ("container pixels"), and the map panes ("div pixels"). We use the map centre for server-side rendering, map top-left for mouse events, and map panes for dynamic rendering.

Reply:
@stephenfarrar Great work man, I admire your knowledge of the Google Maps api. Do you think that by this change, would be easier to fix #505?